components/engine_rocks/src/properties.rs

```diff
@@ -394,6 +395,7 @@ pub struct MvccPropertiesCollector {
     row_versions: u64,
     cur_index_handle: IndexHandle,
     row_index_handles: IndexHandles,
+    key_mode: KeyMode,
```

Comment:
How about use `GcKeyMode` ? Then there are only two enums and will not be confused with the difference of TiDB & Txn.

Reply:
If use ```GcKeyMode```, it  will cause the problem of circular dependency. So I use ```KeyMode ``` here.
And add a comments to indicate that ```KeyMode::Txn``` is used in both TiDB and Txn scenarios